src/modules/mavlink/streams/BATTERY_STATUS.hpp

```diff
@@ -125,6 +125,17 @@ class MavlinkStreamBatteryStatus : public MavlinkStream
 					}
 				}
 
+				static constexpr int mavlink_cells_ext_max = (sizeof(bat_msg.voltages_ext) / sizeof(bat_msg.voltages_ext[0]));
+
+				for (int cell = mavlink_cells_max; cell < mavlink_cells_max + mavlink_cells_ext_max; cell++) {
+					if (battery_status.connected && (cell < battery_status.cell_count) && (cell < uorb_cells_max)) {
+						bat_msg.voltages_ext[cell - mavlink_cells_max] = battery_status.voltage_cell_v[cell] * 1000.0f;
```

Comment:
Not a blocker, but it probably wouldn't hurt to clamp the output so that if someone exceeds 65.536 mV (uint16_t) it doesn't rollover.